apps/condo/domains/acquiring/access/PaymentsFilterTemplate.js

```diff
@@ -5,16 +5,18 @@
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById, getByCondition } = require('@open-condo/keystone/schema')
 
-const { queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
+const { getEmployedOrganizationsByPermissions } = require('@condo/domains/organization/utils/accessSchema')
 
 async function canReadPaymentsFilterTemplates ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
     if (user.isAdmin) return {}
 
+    const permittedOrganizations = await getEmployedOrganizationsByPermissions(user, [])
+
     return {
-        employee: { organization: { ...queryOrganizationEmployeeFor(user.id) } },
+        employee: { organization: { id_in: permittedOrganizations }, user: { id: user.id }, deletedAt: null },
```

Comment:
Isn't it enough for us to just use `{ organization: { id_in: permittedOrganizations } }`?

In `getEmployedOrganizationsByPermissions` we have already checked that the user is an employee of the organization. Is not it?

Reply:
`employee: { organization: { id_in: permittedOrganizations } }` - give me **all** employees from my organizations
`employee: { organization: { id_in: permittedOrganizations }, user: { id: user.id } }` - give me **all my** employees from my organizations

Reply:
ok my mistake